webservice/model.go

```diff
@@ -1,17 +1,67 @@
 package main
 
-// Person Type (more like an object)
-type Person struct {
+// Player Type (more like an object)
+type Player struct {
 	ID        string   `json:"id,omitempty"`
-	Firstname string   `json:"firstname,omitempty"`
-	Lastname  string   `json:"lastname,omitempty"`
+	Firstname string   `json:"first_name,omitempty"`
+	Lastname  string   `json:"last_name,omitempty"`
+	Email     string   `json:"email,omitempty"`
+	Phone     string   `json:"phone,omitempty"`
 	Address   *Address `json:"address,omitempty"`
+	// CreatedOn Time.time `json:"created_on,omitempty"`
```

Comment:
hmmm

Reply:
I gave up. It is going to stay a string for a bit.